<ui version="4.0" >
 <class>DefineUserScalarsForm</class>
 <widget class="QWidget" name="DefineUserScalarsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Variance of Species</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>34</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonVarNew" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/22x22/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonVarDelete" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/22x22/remove.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>34</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QTableView" name="tableVariance" >
        <property name="editTriggers" >
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Species transport</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonNew" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/22x22/add.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete" >
            <property name="text" >
             <string/>
            </property>
            <property name="icon" >
             <iconset resource="resources_pages.qrc" >:/new/prefix1/icons/22x22/remove.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <widget class="QTableView" name="tableScalars" >
        <property name="editTriggers" >
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>457</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_pages.qrc" />
 </resources>
 <connections/>
</ui>
